plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt

```diff
@@ -95,50 +85,6 @@ internal class BaseTh2PluginTest {
         }
     }
 
-    @Test
```

Comment:
question: Why did you have to remove this test?

Reply:
Because LicenseBundleNormalizer doesn't contain a field contained path to normalize file in the new version of LicenseReport. I decided to move this check into integration test

Reply:
Okay, got it